wcivf/apps/elections/templates/elections/includes/_registration_details.html

```diff
@@ -15,58 +15,45 @@ <h2 id='where'>
     https://www.gov.uk/register-to-vote
   </a>
 </p>
-<p>Register before midnight on {{ postelections.first.registration_deadline }} 
-  to vote in the {{ postelections.first.election.name }} on 
-  {{ postelections.first.election.election_date }}.
+<p>Register before midnight on {{ postelection.registration_deadline }} 
+  to vote in the {{ postelection.election.name }} on 
+  {{ postelection.election.election_date }}.
 </p>
-<p>Register before midnight on {{ postelections.first.postal_vote_application_deadline }} 
-  to vote by post in the {{ postelections.first.election.name }} on 
-  {{ postelections.first.election.election_date }}.</p>
+<p>Register before midnight on {{ postelection.postal_vote_application_deadline }} 
+  to vote by post in the {{ postelection.election.name }} on 
```

Comment:
🆘 I'm trying to check for mayoral elections with `is_mayoral` within this line but can't seem to get the syntax correct. Something like: 

`if postelection.is_mayoral: postelection.election.name + "election" else: postelection.election.name`

Reply:
Instead of using `is_mayoral` you can probably get what you need by using [`postelection.friendly_name` ](https://github.com/DemocracyClub/WhoCanIVoteFor/blob/e7993fc056447dd3a055839d9de36d57b68db497/wcivf/apps/elections/models.py#L363)- this will then build a string taking account the type of election it is

Reply:
Although judging from latest comments if we are removing the election name then ignore the above! But its worth taking a look at that method anyway as whenever you want to use the name of the election from a ballot, its that method that you will probably want to use rather than build it out yourself